tap_facebook/__init__.py

```diff
@@ -193,6 +193,11 @@ def advance_bookmark(state, tap_stream_id, bookmark_key, date):
 class InsightsJobTimeout(Exception):
     pass
 
+def log_insights_retry_attempt(details):
+    e = sys.exc_info()[1]
+    LOGGER.info(e.args[0])
```

Comment:
I have no idea what these two lines are doing.

Reply:
Is `e` the exception? If so, I'd call it that.

Also, is it possible for those array indexes to be invalid? It wouldn't be good to throw in the middle of a retry.